src/ts/client/grapesjs/keymaps.ts

```diff
@@ -69,7 +112,7 @@ export const defaultKms = {
   kmOpenPublish: {
     id: 'general:open-publish',
     keys: 'alt+p',
-    handler: editor => toggleCommand(editor, cmdPublish)
+    handler: openPublish
```

Comment:
The ``publish`` command doesn't change the button's state, hence why I don't use the command directly.

Reply:
Could you fix the publish command instead ?
Or leave it as is but create an issue

It's better than a workaround in the code